theme/static/js/hs_file_browser.js

```diff
@@ -140,13 +142,25 @@ function getFileTemplateInstance(file) {
         file.has_model_instance_aggr_folder + "' data-aggregation-appkey='" + file.aggregation_appkey  + "'>" +
         iconTemplate +
         "<span class='fb-file-name'>" + file.name + "</span>" +
+        "<span class='fb-file-name-ellipsis'>" + fileName + "</span>" +
         "<span class='fb-file-type'>" + file.type + " File</span>" +
```

Comment:
these two spans are going to cause a problem.
When the filenames are long, they will overlap:
![overlap](https://github.com/user-attachments/assets/3cd2a1d8-0c8b-4a1f-90c4-3e5b716268a5)

Also, even when the names are short, the display will be duplicated:
![image](https://github.com/user-attachments/assets/b9e06ab9-fec9-4380-9f68-47c4481636b3)